scanners/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>scanners - serverscanning</title>
+    <link rel="stylesheet" href="/common.css">
+  </head>
+  <body>
+    <div class="header"></div>
+    <div class="container">
+      <div>
+        <h1>Scanners</h1>
+        <p>
+          This is a list of scanners made by SSI members. <a href="/">Return to homepage</a>
+        </p>
+        <div>
+          <h2>Temporum</h2>
+          <p>
+            <a href="https://temporum.org">Website</a><br><br>
+            This is a private Discord bot, it has an in game server browser and a streamsniper.
+          </p>
+        </div>
+        <div>
+          <h2>scanner</h2>
```

Comment:
man im gonna be real i just woke up and i am sick today can someone just do it for me and get chatgpt to write a description https://github.com/n2pm/loremipsum

Reply:
yep, 1 sec